metered/src/common/throughput/tx_per_sec.rs

```diff
@@ -48,7 +56,7 @@ impl<T: Instant> TxPerSec<T> {
         self.count += 1;
     }
 
-    pub fn clear(&mut self) {
+    pub(crate) fn clear(&mut self) {
```

Comment:
Any reason you're restricting the visibility here?

Reply:
See above.